src/test/java/org/folio/rest/api/BoundWithStorageTest.java

```diff
@@ -13,33 +23,30 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import java.net.HttpURLConnection;
-import java.util.List;
-import java.util.UUID;
-import java.util.concurrent.TimeUnit;
-
-import static org.awaitility.Awaitility.await;
-import static org.folio.rest.support.http.InterfaceUrls.holdingsStorageUrl;
-import static org.folio.rest.support.http.InterfaceUrls.instancesStorageUrl;
-import static org.folio.rest.support.http.InterfaceUrls.itemsStorageUrl;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-
 @RunWith(JUnitParamsRunner.class)
 public class BoundWithStorageTest extends TestBaseWithInventoryUtil {
-  static ResourceClient boundWithPartsClient  = ResourceClient.forBoundWithParts(client);
+  static ResourceClient boundWithPartsClient  = ResourceClient.forBoundWithParts(getClient());
 
+  @SneakyThrows
+  @After
+  public void beforeEach() {
+    deleteAllById(boundWithPartsClient);
+    clearData();
```

Comment:
Why isn't removing all bound with parts done in the `clear data` operation?

Reply:
> Why isn't removing all bound with parts done in the `clear data` operation?

This is one of those cases where adding to the `clearData()` causes other tests (in other test classes) to fail.
There is an isolation problem.
I suspect a complete redesign (that would require far more changes than what I made in this PR) is necessary to properly address this isolation problem.

I gave up and added the clearing of bound with data here